### PR TITLE
Add Project Loved: Week of June 17th news post

### PR DESCRIPTION
**Pull request by cl8n**
should be almost ready to go

### CODE REVIEW

File: news/2018-06-17-project-loved-week-of-june-17th.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+layout: post
+title: "Project Loved: Week of June 17th"
+date: 2018-06-17 20:00:00 +0000
+---
+
+[Project Loved](https://osu.ppy.sh/community/forums/120) is back with a fresh new set of candidates for the Loved category! Come check them out, and vote for your favourites!
+
+[![](/wiki/shared/news/banners/project-loved.jpg)](https://osu.ppy.sh/community/forums/120)
+
+Welcome back to [Project Loved](https://osu.ppy.sh/community/forums/120)! This week's round of voting showcases some incredibly difficult beatmaps that are sure to give you a challenge. Across the four gamemodes, we've selected 28 maps, and now it's your turn to decide whether or not these will enter the Loved category. If a map has at least 75% positive votes for osu! or osu!taiko, or 80% for osu!catch or osu!mania, it'll make its way into the [Loved beatmap listing](https://osu.ppy.sh/beatmapsets?s=8)!
```
Comment (TPGPL) [inline news/2018-06-17-project-loved-week-of-june-17th.md]: game modes
these -> those?

File: news/2018-06-17-project-loved-week-of-june-17th.md [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+
+Next week, these thresholds will be changing. osu! and osu!taiko will be increasing to 80%, following feedback that we've gotten from you! Both of these modes will also be cutting their candidate count down to five—[Toy](https://osu.ppy.sh/users/2757689) believes that this will ensure that all five candidates are strong picks, and it'll make sure that there's no shortage of awesome maps in the future.
```
Comment (TPGPL) [inline news/2018-06-17-project-loved-week-of-june-17th.md]: spaces before and after hyphen?

File: news/2018-06-17-project-loved-week-of-june-17th.md [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+- [osu!mania](#osumania)
+
+## <a name="osu" id="osu"></a>osu!
+
+[![](/wiki/shared/news/2018-06-17-project-loved-week-of-june-17th/osu/toaru-shoukoku-no-ohimesama.jpg)](https://osu.ppy.sh/community/forums/topics/762191)
+
+[nao - Toaru Shoukoku no Ohimesama](https://osu.ppy.sh/beatmapsets/370605) by [Settia](https://osu.ppy.sh/users/2445897)  
+*written by [Kaifin](https://osu.ppy.sh/users/2596942)*
+
+Toaru Shoukoku no Ohimesama is the quintessential [Settia](https://osu.ppy.sh/users/2445897) map, featuring ridiculous cross screen jumps overtop of a bombastic eroge song. If you aren't up to the challenge of the 8 star top difficulty, there are 4 lower diffs that vary in difficulty for you to enjoy! It's got some jumps, yo.
```
Comment (TPGPL) [inline news/2018-06-17-project-loved-week-of-june-17th.md]: Settia's

File: news/2018-06-17-project-loved-week-of-june-17th.md (hunk 2 above)
Comment (cl8n) [reply news/2018-06-17-project-loved-week-of-june-17th.md]: it's an em dash, not a hyphen. I don't think you're supposed to put spaces around them

File: news/2018-06-17-project-loved-week-of-june-17th.md (hunk 3 above)
Comment (cl8n) [reply news/2018-06-17-project-loved-week-of-june-17th.md]: the name is acting as an adjective here, not possessive

File: news/2018-06-17-project-loved-week-of-june-17th.md [hunk 4]
```diff
@@ -145,12 +145,12 @@ Here we go again! Nekosogi Destruction is a speed map that will fool you from it
 
 ## <a name="osucatch" id="osucatch"></a>osu!catch
 
-[![](/wiki/shared/news/2018-06-17-project-loved-week-of-june-17th/catch/furioso-melodia.jpg)](https://osu.ppy.sh/community/forums/topics/762214)
+[![](/wiki/shared/news/2018-06-17-project-loved-week-of-june-17th/catch/gigadelic-camellia-s-the-tera-rmx-.jpg)](https://osu.ppy.sh/community/forums/topics/762314)
 
-[gmtn. (witch's slave) - furioso melodia](https://osu.ppy.sh/beatmapsets/603435) by [Minato Yukina](https://osu.ppy.sh/users/531253)  
-*written by [SartNYAN OwO xD](https://osu.ppy.sh/users/4100941)*
+[Camellia - gigadelic (Camellia's "The TERA" RMX)](https://osu.ppy.sh/beatmapsets/388122) by [Alumetorz](https://osu.ppy.sh/users/531253)
```
Comment (Joehuu) [inline news/2018-06-17-project-loved-week-of-june-17th.md]: Forgot the other mapper.

File: news/2018-06-17-project-loved-week-of-june-17th.md [hunk 5]
```diff
@@ -0,0 +1,245 @@
...
+[![](/wiki/shared/news/2018-06-17-project-loved-week-of-june-17th/osu/kyouki-chinden.jpg)](https://osu.ppy.sh/community/forums/topics/762199)
+
+[Yousei Teikoku - Kyouki Chinden](https://osu.ppy.sh/beatmapsets/129520) by [Saten](https://osu.ppy.sh/users/444506) and [OnosakiHito](https://osu.ppy.sh/users/290128)  
+*written by [Toy](https://osu.ppy.sh/users/2757689)*
+
+[Saten](https://osu.ppy.sh/users/444506)'s style has become so distinct, that I'm sure anyone even remotely familiar with his style could point out one of his maps with their eyes closed. Kyouki Chinden is no exception, and whether you enjoy singletapping 280BPM or alternating 140BPM, this map fits exactly what you'd expect from a Saten Mirai Nikki mapset.
+
+[![](/wiki/shared/news/2018-06-17-project-loved-week-of-june-17th/osu/disperagioia.jpg)](https://osu.ppy.sh/community/forums/topics/762200)
+
+[gmtn. (witch's slave) - disperagioia](https://osu.ppy.sh/beatmapsets/566624) by [Dokito](https://osu.ppy.sh/users/537084)
```
Comment (Joehuu) [inline news/2018-06-17-project-loved-week-of-june-17th.md]: `Dokito` -> `DoKito`

File: news/2018-06-17-project-loved-week-of-june-17th.md [hunk 6]
```diff
@@ -0,0 +1,245 @@
...
+
+The chart was also used back in the osu!mania World Cup 2016, a time when long notes and anti-meta was only at the start of its rise in the competitive scene. Given the playstyle's extreme growth over the years following this tournament, and the previous nomination of such charts as Voyage 1970 and Mirage, it only makes sense to add Morgenglut alongside as a nomination to the loved section.
+
+[![](/wiki/shared/news/2018-06-17-project-loved-week-of-june-17th/mania/katayoku-no-tori.jpg)](https://osu.ppy.sh/community/forums/topics/762221)
+
+[Hommarju feat. mamiko - Katayoku no Tori](https://osu.ppy.sh/beatmapsets/237512) by [Awang411](https://osu.ppy.sh/users/1014938) and [Gexigamma](https://osu.ppy.sh/users/3149376)
```
Comment (Joehuu) [inline news/2018-06-17-project-loved-week-of-june-17th.md]: `no Tori` -> `No Tori`

Correct other instances.

**Pull request merged**